content/300-guides/400-migrate-to-prisma/03-migrate-from-mongoose.mdx

```diff
@@ -0,0 +1,1056 @@
+---
+title: 'Migrate from Mongoose'
+metaTitle: 'How to migrate from Mongoose to Prisma'
+metaDescription: 'Learn how to migrate from Mongoose to Prisma'
+---
+
+<TopBlock>
+
+This guide describes how to migrate from Mongoose to Prisma. It uses an extended version of the [Mongoose Express example](https://github.com/Automattic/mongoose/tree/master/examples/express) as a [sample project](https://github.com/prisma/migrate-from-mongoose-to-prisma) to demonstrate the migration steps. You can find the example used for this guide on [GitHub](https://github.com/prisma/migrate-from-mongoose-to-prisma).
+
+This migration guide uses MongoDB as the example database.
```

Comment:
Is there another option?

Reply:
Could you clarify what you mean?

Do you mean the Mongoose example starter?

Reply:
"This migration guide uses MongoDB as the example database." indicates that you could also use another database. But I don't think there is another option for Mongoose than to use MongoDB.